zerver/lib/bugdown/__init__.py

```diff
@@ -1160,7 +1160,7 @@ def do_convert(md, realm_domain=None, message=None):
         cleaned = _sanitize_for_log(md)
 
         # Output error to log as well as sending a zulip and email
-        logging.getLogger('').error('Exception in Markdown parser: %sInput (sanitized) was: %s'
+        logging.error('Exception in Markdown parser: %sInput (sanitized) was: %s'
```

Comment:
Oh.  I think I know what this is for.  I'm pretty sure it's important, and exists to prevent the `logging.error` from being caught by the AdminZulipHandler error reporting system and putting the invalid input into a Zulip message.